### DISCUSSION TITLE
Would it be possible to add a feature to auto-show attack modifiers when players use them?

### DISCUSSION THREAD

**Discussion by NimecShady** [Lurkars/gloomhavensecretariat#323] [Ideas]
Hey there,

We have been using your program for months now and it is fantastic. It's always interesting when we fire it up and it's obvious that new features and changes have been made. It just keeps getting better all the time. 

We play in a group of three local with one remote. To simplify things for the remote player, we are letting him draw his attack modifier cards through the application while we locally flip our real cards over. 

One thing that we noticed though is every time our remote player attacks we have to make sure we click on the attack icon on his character bar to be able to see the card that gets flipped. It's not really a big deal but a lot of times we forget to hit it and it's fun to be able to reveal the card and everyone react at the same time. 

I was just wondering if it was possible to add a feature or an option that if a player draws an attack modifier  card that the bar automatically flips to and shows the card as if you had clicked it on prior. That way whenever a player draws an attack card remotely from a different client all users could see the card being flipped in real time. 

Maybe this isn't a good design choice for the program, I'm not really sure. It's just something we noticed that we miss a lot and it would be cool if there were an option just to simply show automatically when the remote player draws an attack modifier card. 

Cheers bud, the program is fantastic.

**Comment by Lurkars** [comment]
Use the setting `Permanent Character AM deck`. When enable a character's deck it's then always visible under the character. This must be done for every clients who wants to always see the deck.
